_includes/sidebar.html

```diff
@@ -10,113 +10,113 @@
   {% for category in site.data.documentsupdated %} {% if forloop.last == true %}
   <li class="folder topfolder lowerimage">
     <a class="highlightlink" href="">
-      <span class="sidebarbutton"><span></span></span>
-      <span> {{ category.categoryname }} </span></a
-    >
+      <span class="sidebarbutton">
+        <span></span>
+      </span>
+      <span> {{ category.categoryname }}</span>
```

Comment:
Please remove the space between “<span>” and “{{”.

(Only asking for this because you removed one before “</span>”.)